File: pangram/JS/pangram.js
const pangram = string => {
const alphabet = "abcdefghijklmnopqrstuvwxyz".split("");
const check = string.toLowerCase();

for (const letter of alphabet) {
  if (check.includes(letter)) continue;
  else return "not pangram";
}
return "pangram";
}

module.exports = pangram;

File: pangram/Java/Pangram.java
public class Pangram {
    public static String pangrams(String s) {
        String[] alphabet = "abcdefghijklmnopqrstuvwxyz".split("");
        s = s.toLowerCase();
        for (String i : alphabet) {
            if (s.contains(i)) continue;
            else return "not pangram";
        }
        return "pangram";
    }
}
